### PR TITLE
Fix deltaR calculation in HLT widejet module

### PR DESCRIPTION
**Pull request by duanders**
Modifies HLTFatJetMassFilter so that it computes DeltaR correctly.  (The current behavior does not treat DeltaPhi modulo 2 pi.)

### CODE REVIEW

**Comment by fwyzard** [comment]
please test

**Comment by Martin-Grunewald** [comment]
need a 81X PR as well!

File: HLTrigger/JetMET/src/HLTFatJetMassFilter.cc [hunk 1]
```diff
@@ -128,11 +129,11 @@ HLTFatJetMassFilter<jetType>::hltFilter(edm::Event& iEvent, const edm::EventSetu
   
   // apply radiation recovery
   for ( recojet = recojets.begin() ; recojet != recojets.end() ; recojet++) {
-    double DeltaR1 = sqrt(pow(recojet->phi()-j1.phi(), 2.)+pow(recojet->eta()-j1.eta(),2.));
-    double DeltaR2 = sqrt(pow(recojet->phi()-j2.phi(), 2.)+pow(recojet->eta()-j2.eta(),2.));
-    if(DeltaR1 < DeltaR2 && DeltaR1 < fatJetDeltaR_) {
```
Comment (fwyzard) [inline HLTrigger/JetMET/src/HLTFatJetMassFilter.cc]: @duanders from the code, I think the bug you are trying to fix is that |phi2 - phi1| does not properly wrap around π , right ?

**Comment by fwyzard** [comment]
+1

**Comment by fwyzard** [comment]
I think this should be simple and safe enough to enter directly as bugfix in 8.0.x - @davidlange6 what do you say ?

File: HLTrigger/JetMET/src/HLTFatJetMassFilter.cc (hunk 1 above)
Comment (duanders) [inline HLTrigger/JetMET/src/HLTFatJetMassFilter.cc]: @fwyzard yes that's right.

**Comment by davidlange6** [comment]
it can go into 80x, but would like an 81x request too. (also for the second request from @duanders )

**Pull request merged**